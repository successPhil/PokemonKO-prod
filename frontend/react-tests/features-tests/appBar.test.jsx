import {expect, test} from "vitest"
import ResponsiveAppBar from "../../src/features/AppBar/AppBar";

test("App Bar renders without crashing", ()=> {
    const component = <ResponsiveAppBar />;
    expect(component).toMatchSnapshot();
})