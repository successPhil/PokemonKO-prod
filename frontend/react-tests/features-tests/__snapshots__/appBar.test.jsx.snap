// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App Bar renders without crashing 1`] = `<ResponsiveAppBar />`;
